### DISCUSSION TITLE
Mingda Magician Max When pressing the stop button Error

### DISCUSSION THREAD

**Discussion by AlhambraFahrer** [OrcaSlicer/OrcaSlicer#5157] [General]
Hello,

maybe someone can help me.

I have been working with Cura and now want to test Orca Slicer.

The printer is a Mingda Magician Max (Marlin firmware)

There is no suitable profile for it in Orca Slicer.

I have therefore started to create one myself.

I have already been able to make a few test prints.

However, when I press the stop button on the printer (LCD screen) to cancel the test, the printer crashes with the following error message:

Printer stopped Please reset Homing failed.

G Codes files created with Cura do not have this problem.
If the stop button is pressed, the printer aborts and returns to the home position.

I would be grateful for any tips.
![IMG_20240426_123300_copy_1280x720](https://github.com/SoftFever/OrcaSlicer/assets/104303110/57ae8c75-5b22-41a8-a4d1-dbb0eefcc5dc)

**Comment by falkenhawk** [comment]
How did you create the profile? Did you use ender v3 or any other profile as a base which you then modified?

**Comment by AlhambraFahrer** [comment]
Thank you for your help.

Since the printer is not selectable, I used the "Generic Marlin Printer" option during setup.

Then I had to adjust the speeds and acceleration as they were set far too high.

The printer can't cope with that.

Apart from a few exceptions, printing works fine. 

But on the one hand the error when canceling the print and the erroneous prints that still occur out of nowhere.

I haven't found a solution yet.

At the moment I have integrated Octoprint and entered a stop G-code.
That works .

But not directly on the printer display.

Here are a few prints with Orca Slicer. Sometimes ok and sometimes faulty.


![1715274216376](https://github.com/SoftFever/OrcaSlicer/assets/104303110/bf85e232-36c5-48c2-982f-40bea328e6fe)
![1715274216385](https://github.com/SoftFever/OrcaSlicer/assets/104303110/88a03cac-ec91-4da9-bf7c-3f07370e4ead)
![1715274216392](https://github.com/SoftFever/OrcaSlicer/assets/104303110/3436e3a4-7b0e-46e8-9a68-144409574393)
![1715274216399](https://github.com/SoftFever/OrcaSlicer/assets/104303110/b988691b-4a99-4c29-ad76-b83ef1d7a4a2)
